src/pages/gen2/build-a-backend/data/data-modeling/relationships/index.mdx

```diff
@@ -220,78 +284,109 @@ You can make a "has one" or "has many" relationship bidirectional with the `belo
 
 ```typescript
 const schema = a.schema({
-  Project: a.model({
+  Customer: a.model({
     name: a.string(),
-    team: a.hasOne('Team')
+    activeCart: a.hasOne('Cart')
   }),
-  Team: a.model({
-    name: a.string(),
-    project: a.belongsTo('Project'),
+  Cart: a.model({
+    items: a.string().array(),
+    customer: a.belongsTo('Customer'),
   }),
-});
+}).authorization([a.allow.public()]);
 ```
 
+<BlockSwitcher>
+<Block name="Customer -> Cart request">
+
+![Bidirectional @hasOne: Customer -> Cart request](/images/gen2/data-modeling/has-one-customer-to-cart.png)
+
+</Block>
+<Block name="Cart -> Customer request">
+![Bidirectional @hasOne: Cart -> Customer request](/images/gen2/data-modeling/has-one-cart-to-customer.png)
+</Block>
+
+</BlockSwitcher>
+
 ```ts
 const client = generateClient<Schema>();
 
-const { data: team } = await client.models.Team.get({ id: 'FABF27EE-25D2-4851-AE1C-09BCA1604C77' });
+const { data: customer } = await client.models.Customer.get({
+  id: 'FABF27EE-25D2-4851-AE1C-09BCA1604C77'
+});
 
-console.log((await team.project()).data?.name); // There is a 'project' field in the team
+const { data: cart } = await customer.activeCart();
 
-const { data: teamProject } = await team.project();
-const { data: teamProjectRelatedTeam } = await teamProject.team();
-// teamProjectRelatedTeam is equivalent to original team
+// references back to the same customer as above
+const { data: belongsToCustomer } = await cart.customer();
```

Comment:
It should be clear from context, but do you want an explicit eager/lazy loading example in the bidi examples?